sw/device/lib/base/bitfield.h

```diff
@@ -7,7 +7,6 @@
 
 #include <stdint.h>
 
-// Header Extern Guard  (so header can be used from C and C++)
```

Comment:
Don't blanket remove these comments - they're there because contributors were asking what the guards were for.

Reply:
Done.